capstone/src/main/webapp/youtubeScript.js

```diff
@@ -108,6 +112,12 @@ function getAttributeTotals(attributeScores) {
   const attributeTotals = new Map();    
   for (let i = 0; i < requestedAttributes.length; i++) {
     for (let j = 0; j < attributeScores.length; j++) {
+      // populates attributeData to support CSV output and attributeTotals to support averaging
+      if(attributeData[j] == null) {
+        attributeData[j] = [(attributeScores[j].attributeScores[requestedAttributes[i]].summaryScore.value)];
```

Comment:
It might be useful to create local variables for `attributeScores[requestedAttributes[i]].summaryScore.value` and `attributeScores[j]` to improve readability in lines 117 to 124.

Reply:
addressed in "improve readability"